src/OmniSharp/Api/Intellisense/OmnisharpController.Intellisense.cs

```diff
@@ -104,11 +104,17 @@ private IEnumerable<AutoCompleteResponse> MakeSnippetedResponses(AutoCompleteReq
 
         private AutoCompleteResponse MakeAutoCompleteResponse(AutoCompleteRequest request, ISymbol symbol, bool includeOptionalParams = true)
         {
+            var displayNameGenerator = new SnippetGenerator();
+            displayNameGenerator.IncludeMarkers = false;
+            displayNameGenerator.IncludeOptionalParameters = includeOptionalParams;
+
             var response = new AutoCompleteResponse();
             response.CompletionText = symbol.Name;
 
             // TODO: Do something more intelligent here
-            response.DisplayText = symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
+            response.DisplayText = symbol.Kind == SymbolKind.Method ?
+                displayNameGenerator.Generate(symbol) :
```

Comment:
Is this conditional necessary? Can't you always use displayNameGenerator.Generate?